packages/slate/src/changes/with-schema.js

```diff
@@ -121,12 +104,22 @@ function normalizeNodeAndChildren(change, node, schema) {
  * @return {Node}
  */
 
-function refindNode(change, node) {
+function refindNode(change, node, lastPath) {
```

Comment:
Can you change this variable to `previousPath`?

Reply:
Yes, I will change it.